src/collection.js

```diff
@@ -797,7 +797,7 @@ export default class Collection {
    */
   loadDump(records) {
     const reject = msg => Promise.reject(new Error(msg));
-    if (!(records instanceof Array)) {
+    if (!(typeof records.sort == 'function')) {
```

Comment:
`Array.isArray(records)` maybe? Or are you suggesting passed arg is possibly not a *true* array, like `Arguments` or `NodeList`?